spec/features/auth/login_feature_spec.rb

```diff
@@ -109,6 +109,36 @@ module CleanRoom
     expect(page).to have_current_path(new_user_session_path)
   end
 
+  it "Login form is enabled when local_login is enabled" do
+    APP_CONFIG["local_login"] = { "enabled" => true }
```

Comment:
The default is enabled, so you don't need to do this here.

Reply:
I prefer to keep this very line. The config should be explicitly specified for this test case, for whatever default value is.